### PR TITLE
Grammatical updated README.md file\n and added Snake-game

### PR DESCRIPTION
**Pull request by Shadow-max12**
Updated **README.md** file.
Added Snake Game in **What Games Will we Build**.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ Code samples and other handouts for our course. See the [**practice exercises**]
 
 ## Course Summary
 
-Learning to program can be overwhelming. Concepts and facts come at you fast and most courses don't cover many of them at all or at a beginner's pace. **This is not most courses**. Python for absolute beginners is our premier course for beginning developers. We start at the very beginning, teaching you the big ideas and concepts covered in a CS 101 course. Then we move on to writing increasingly complex code and applications in Python.
+Learning to program can be overwhelming. Concepts and facts come at you fast and most courses don't cover many of them at all or at a beginner's pace. **This is not like most courses**. Python for absolute beginners is our premier course for beginning developers. We start at the very beginning, teaching you the big ideas and concepts covered in a CS 101 course. Then we move on to writing increasingly complex code and applications in Python.
```
Comment (mikeckennedy) [inline README.md:11]: Hi, I know it's not the most common way of speaking, but I did want it to say: **This is not most courses**, to emphasize what was said in the prior sentence.

File: README.md [hunk 2]
```diff
@@ -95,6 +95,7 @@ Python course would cover.
 2. **TIC-TAC-TOE**
 3. **Rock-Paper-Scissors (3-way and 7-way)**
 4. **A Connect4 clone**
+5. **Snake Game**
```
Comment (mikeckennedy) [inline README.md:98]: We didn't cover a snake game did we? What part are you referring to?

File: README.md (hunk 1 above)
Comment (Shadow-max12) [reply README.md:11]: Alright,it's up to you.Thanks for review.

File: README.md (hunk 2 above)
Comment (Shadow-max12) [reply README.md:98]: Actually I am referring to adding **Snake Game** in this Course.So it'll cover basic and intermediate Concept.

**Pull request closed without merging**